.circleci/config.yml

```diff
@@ -90,21 +84,60 @@ jobs:
             bundle exec rake knapsack:rspec
             yarn test
             bundle exec slim-lint app/views
-
       - run:
           name: Code Climate Test Coverage
           command: |
             ./cc-test-reporter format-coverage -t simplecov -o "coverage/codeclimate.$CIRCLE_NODE_INDEX.json"
             aws s3 sync coverage/ "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM"
-
       # collect reports
       - store_test_results:
           path: /tmp/test-results
       - store_artifacts:
           path: /tmp/test-results
           destination: test-results
-
       - deploy:
           command: |
             aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
             ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
+  build-latest-container:
+    working_directory: ~/identity-idp
+    docker:
+      - image: circleci/ruby:2.6
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          rev=$(git rev-parse --short HEAD)
+          docker build -t logindotgov/idp:latest -t logindotgov/idp:"${rev}" -f production.Dockerfile .
+          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
+          docker push logindotgov/idp:"${rev}"
+          docker push logindotgov/idp:latest
+  build-release-container:
+    working_directory: ~/identity-idp
+    docker:
+      - image: circleci/ruby:2.6
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: |
+          if [ -z "$CIRCLE_TAG" ]; then exit 0; fi # this shouldn't be necessary...
+          docker build -t logindotgov/idp:$CIRCLE_TAG -f production.Dockerfile .
+          echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
+          docker push logindotgov/idp:$CIRCLE_TAG
+workflows:
+  version: 2
+  release:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: "/.*/"
+      - build-latest-container:
+          requires:
+            - build
+      - build-release-container:
+          requires:
+            - build
+          filters:
+            tags:
+              only: "/^[0-9]{4}-[0-9]{2}-[0-9]{2,}.*/"
```

Comment:
same question here: does this only work on like UUID formats? Can/should we comment it?

Reply:
This is based off your `2020-01-01T123456` tags (`git tags -l`).

It would be good if the release tags could move to a semver format (`/v1.*/`), much easier for humans to understand, but this should integrate with whatever the current release process is without any changes from your release process.

Reply:
Ok can we add a comment with an example timestamp to make it easier to understand?

Reply:
Also this seems backwards, we only tag releases with that timestamp *after* we've merged to production branch.... but we've already deployed artifacts to lower environments like `staging` before that.... maybe I'm not understanding the expected workflow here